primitives/block/src/micro_block.rs

```diff
@@ -63,9 +63,13 @@ impl MicroBlock {
 
     /// Returns the available size, in bytes, in a micro block body for transactions.
     pub fn get_available_bytes(num_equivocation_proofs: usize) -> usize {
-        Policy::MAX_SIZE_MICRO_BODY
-            - (/*equivocation_proofs vector length*/2 + num_equivocation_proofs * EquivocationProof::MAX_SIZE
-            + /*transactions vector length*/ 2)
+        #[allow(clippy::identity_op)]
+        Policy::MAX_SIZE_MICRO_BODY.saturating_sub(
+            0
+            + /* equivocation_proofs vector length */ 2
+            + num_equivocation_proofs * EquivocationProof::MAX_SIZE
+            + /* transactions vector length */ 2,
```

Comment:
It's hard to me to read it, and do we really need to add 0?
```suggestion
               2 // equivocation_proofs vector length
            + num_equivocation_proofs * EquivocationProof::MAX_SIZE
            + 2 // transactions vector length,
```

Reply:
It's no longer there in the newest version. I think I formatted it in this way because everything else was botched after running `cargo fmt` on it.

Reply:
It's no longer there in the newest version. I think I formatted it in this way because everything else was botched after running cargo fmt on it.